### ISSUE TITLE
Embedded arrays contain empty string

### ISSUE THREAD

**Issue by messyOne** [stevetarver/excel-as-json#23]
Having excel columns using embedded arrays like *aliases[]* which are empty produce an array with an empty string. So 

`aliases[]: stormagedden;bob` 
creates 
```
"aliases": [
    "stormagedden",
    "bob"
  ]
```
which is fine.

`aliases[]:` 
creates 
```
"aliases": [
    ""
  ]
```
which should be 
```
"aliases": [ ]
```

**Comment by pasine** [comment]
I'm also facing the same issue. I see there is an option `omitEmptyFields` to completely hide empty fields, but this is not what I'm looking for.
Is there a specific reason why you cannot return 
```
{
key: null,
aliases[]: []
}
```
instead of

```
{
key: "",
aliases[]: [""]
}
```

**Comment by stevetarver** [comment]
Fixed in v2.0.2.

Thanks for your bug reports.
